deploy/development.sh

```diff
@@ -8,8 +8,6 @@ kubectl delete deployments.apps localstack -n jvm-build-service
 DIR=`dirname $0`
 kubectl apply -f $DIR/namespace.yaml
 kubectl config set-context --current --namespace=jvm-build-service
-find $DIR -name maven-v0.2.yaml -exec sed -i s/redhat-appstudio/${QUAY_USERNAME}/ {} \;
-find $DIR -name maven-v0.2.yaml -exec sed -i s/3dcd8603061c7ab2cedfe128c80cac9726721a74/dev/ {} \;
```

Comment:
that said, I'm still not clear how the specific sha tag will get changed to `dev` instead

I'll check out your PR today and discover one way or the other

Reply:
ok I finally found the `kustomization.yaml` file that was changing this ... not sure why I was having such a hard time finding this the other day